packages/app/src/app/pages/NewDashboard/Components/Sandbox/index.tsx

```diff
@@ -239,6 +239,8 @@ const GenericSandbox = ({ isScrolling, item }: GenericSandboxProps) => {
   };
 
   const sandboxProps = {
+    prNumber: item.sandbox.prNumber,
+    originalGit: item.sandbox.originalGit,
```

Comment:
Aren't these props not on `sandbox` itself? (on line 249)

Reply:
But the sandbox isn't passed in totality down

Reply:
It is, on line 251 the `item.sandbox` is passed down in totality. Or is there another reason to send down these props separately?

Reply:
ohh, my bad, used that instead